VideoApp/Video-CommunityTests/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.1.8</string>
```

Comment:
These version changes were done when I was testing CI but I'll just leave them in. After this PR is merged I will create a new release for 0.1.9.